com.unity.UnityHub.yaml

```diff
@@ -30,6 +30,14 @@ add-extensions:
   org.freedesktop.Platform.Compat.i386.Debug:
     directory: lib/debug/lib/i386-linux-gnu
     version: '22.08'
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: .
+  org.freedesktop.Platform.ffmpeg_full.i386:
+    directory: lib32/ffmpeg
+    version: '22.08'
+    add-ld-path: .
```

Comment:
I don't think we need the 32-bit version. From #94, Unity invokes whatever `ffmpeg` is in the path, and the one in the path is the 64-bit version at `/usr/bin/ffmpeg`.

Plus, the given `directory`, `lib32/ffmpeg`, does not exist in `/app`. So at best, defining this extension has no effect; and at worst, if the extension is installed the app may fail to start. (I don't know what Flatpak does if the target directory for an extension is missing.)

```suggestion
```